doc/api/events.md

```diff
@@ -266,7 +266,7 @@ but important side effect: any *additional* listeners registered to the same
 listener that is in the process of being added.
 
 ```js
-const myEmitter = new MyEmitter();
+const myEmitter = new EventEmitter();
```

Comment:
Instead of just replacing the value, what about adding the part right on top?

```suggestion
class MyEmitter extends EventEmitter {}

const myEmitter = new MyEmitter();
```

That way the code block is almost possible to run without any further changes while keeping the original intent. The same applies to the other part.